docs/docs/guides/testing.md

```diff
@@ -77,6 +77,87 @@ describe('The form state should', () => {
 });
 ```
 
+## Testing Recoil state with asyncronous default inside of a React component
+
+A common pattern for the default state of atoms is to have an asynchronous function fetch the default state of an atom. However, while testing, the component is immediately suspended upon mounting and will not update in the dom without acting. To test this scenario, we need a helper function:
```

Comment:
This pattern isn't just for async atom defaults, but it useful for testing any async dependency in Recoil, such as async selectors or effects as well.

Reply:
How should I best describe this in the summary? 

I was thinking something like:

> A common pattern for atoms is using asynchronous queries fetch the state of the atom, selector, or effect. This causes the component to be suspended. However, while testing, the component is immediately suspended upon mounting and will not update in the DOM without acting. To test this scenario, we need a helper function:

Reply:
Or maybe `"It is common for atoms and selectors to have asynchronous dependencies or queries.  This can cause components to be suspended during initial rendering when testing.  When using Jest, you should use `act()` to wait for the promises to resolve and update the DOM.  Here is an example helper function:"`